desktop/apps/miami_fair_week/components/MiamiFairWeekPage.js

```diff
@@ -0,0 +1,166 @@
+              </Text>
+            </Col>
+            <Col lg={5} md={12} sm={12} xs={12} style={{ marginBottom: 25 }}>
+              <Text textSize='medium' color={colors.graySemibold}>
+                Public Viewing<br />
+                December 7 12:00pm–6:00pm<br />
+                The Bath Club<br />
+                5937 Collins Ave, Miami Beach<br />
+              </Text>
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+
+      <Row>
+        <Col lg={4} md={4} sm={12} xs={12}>
+          <SectionTitle titleSize="large">
+            Stories from Miami
+          </SectionTitle>
+        </Col>
+        <Col lg={8} md={8} sm={12} xs={12}>
+          <Row style={{ marginBottom: 25 }}>
+            <Col lg={7} md={7} sm={6} xs={12}>
+              <img style={{ marginBottom: 10, width: '100%' }} src='https://d7hftxdivxxvm.cloudfront.net/?resize_to=width&src=https%3A%2F%2Fartsy-media-uploads.s3.amazonaws.com%2FlhQti2pXrsXDLlS8jJEIDg%252F_AR_3443.jpg&width=1100&quality=95' />
+            </Col>
+            <Col lg={5} md={5} sm={6} xs={12}>
+              <Title titleSize='small' style={{ margin: '0 0 5px', lineHeight: 1 }}>
+                The 20 Best Booths at Art Basel in Miami Beach
+              </Title>
+              <Text textStyle='primary' textSize='small'>
+                ANNA LOUIE SUSSMAN
+              </Text>
+            </Col>
+          </Row>
+          <Row style={{ marginBottom: 25 }}>
+            <Col lg={7} md={7} sm={6} xs={12}>
+              <img style={{ marginBottom: 10, width: '100%' }} src='https://d7hftxdivxxvm.cloudfront.net/?resize_to=width&src=https%3A%2F%2Fartsy-media-uploads.s3.amazonaws.com%2FvClMRePyeu9nCashzAgEeA%252F_AR_3326.jpg&width=1100&quality=95' />
+            </Col>
+            <Col lg={5} md={5} sm={6} xs={12}>
+              <Title titleSize='small' style={{ margin: '0 0 5px', lineHeight: 1 }}>
+                50 Must-See Artworks at Miami Art Week’s Satellite Fairs
+              </Title>
+              <Text textStyle='primary' textSize='small'>
+                Artsy Editorial
+              </Text>
+            </Col>
+          </Row>
+          <Row style={{ marginBottom: 25 }}>
+            <Col lg={7} md={7} sm={6} xs={12}>
+              <img style={{ marginBottom: 10, width: '100%' }} src='https://d7hftxdivxxvm.cloudfront.net/?resize_to=width&src=https%3A%2F%2Fartsy-media-uploads.s3.amazonaws.com%2FOX8QZ5TCXVt8szczr7oWZQ%252Fammann.jpg&width=1100&quality=95' />
+            </Col>
+            <Col lg={5} md={5} sm={6} xs={12}>
+              <Title titleSize='small' style={{ margin: '0 0 5px', lineHeight: 1 }}>
+                The 10 Best Booths at Design Miami/
+              </Title>
+              <Text textStyle='primary' textSize='small'>
+                Artsy Editorial
+              </Text>
```

Comment:
I'm a big fan of this approach -- sweeping over with static content and then iterating / componentizing (only) when necessary. It's often the case where breaking things apart from the get-go leads to premature optimization.